docs/dev-concepts/start-building.md

```diff
@@ -128,6 +126,56 @@ Consider following these best practices when developing your app:
     - \# of active conversations: Conversations with at least one message
     - \# of returning conversations: Conversations with at least one message per participant
 
+## Performance best practices
+
+Follow these guidelines to optimize your app’s performance.
+
+### For JavaScript
+
+Using the JavaScript client SDK (`xmtp-js`):
+
+- [Upgrade to ≥v9.0.0](https://github.com/xmtp/xmtp-js/releases/tag/v9.0.0), which provides a 10x smaller web bundle size.
+- [Cache conversations](/docs/client-sdk/javascript/tutorials/quickstart#cache-conversations)
+- [Page through messages](/docs/client-sdk/javascript/tutorials/quickstart#list-messages-in-a-conversation-with-pagination)
+- [Compress messages](https://xmtp.org/docs/client-sdk/javascript/tutorials/quickstart#compression)
+- [Serialize decoded messages](https://github.com/xmtp/xmtp-js/releases/tag/v8.0.0)
```

Comment:
Should we update the xmtp-js README to provide section and code example for this functionality?